notebooks/scikit-learn/Training and Prediction with scikit-learn.ipynb

```diff
@@ -208,15 +220,19 @@
     "])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Export the model to a file"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# ---------------------------------------\n",
-    "# Export the model to a file\n",
-    "# ---------------------------------------\n",
     "model = 'model.joblib'\n",
     "joblib.dump(pipeline, model)"
```

Comment:
Should this notebook include a "Prediction" section, and remove "Next Steps:" at the end?